dotnet/src/Skills/Skills.OpenAPI/OpenApi/OpenApiDocumentParser.cs

```diff
@@ -186,7 +186,7 @@ private static List<RestApiOperation> CreateRestApiOperations(string? serverUrl,
                 string.IsNullOrEmpty(serverUrl) ? null : new Uri(serverUrl),
                 path,
                 new HttpMethod(method),
-                operationItem.Description,
+                string.IsNullOrEmpty(operationItem.Description) ? operationItem.Summary : operationItem.Description,
```

Comment:
Use the null-coalescing operator ??

Reply:
@markwallace-microsoft I think it's not going to work if `operationItem.Description` is not null, but empty string.

Reply:
It won't compile because the `string.IsNullOrEmpty(operationItem.Description)` method returns Boolean while the `operationItem.Summary` property returns string:
<img width="832" alt="image" src="https://github.com/microsoft/semantic-kernel/assets/68852919/bae5c29c-981c-4d70-998e-8068eedf17bb">